docs/scos/dev/upgradability-services/upgradability-reference.md

```diff
@@ -1,19 +1,161 @@
 ---
-title: Upgradability reference
-description: Rerference infomation for evaluator and upgrader tools.
+title: Upgradability guidelines
+description: Reference information for evaluator and upgrader tools.
 last_updated: Nov 25, 2021
 template: concept-topic-template
 ---
 
-## Method is overridden with a Factory, Dependency Provider, Repository, or Entity Manager extended
+## Entity name is not unique
 
-### What is the nature of the upgradability error?
+Names of transfers, transfer properties, database tables, database columns, methods, and constants should be unique. to the extent of making it impossible to accidentally match the name of a core entity introduced in future.
```

Comment:
Names of transfers, transfer properties, database tables, database columns, methods, constants, and any other entities should be unique, to the extent of making it impossible to accidentally match the name of a core entity introduced in the future.